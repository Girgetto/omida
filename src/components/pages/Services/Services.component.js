import React from 'react'
import { useTranslation } from 'react-i18next'
import { SEO } from '../../layout'

const Services = ({ className }) => {
  const [t] = useTranslation()
  return (
    <div className={className}>
      <SEO title="Remida | Services">
        <h1>{t('services.Intro')}</h1>
        <p>{t('services.IntroText')}</p>
        <div className="titles">
          <h2>
            {t('services.LuxuryRealEstate')} | {t('services.BusinessAdvisory')}{' '}
            |{t('services.TechnicalAdvice')} | {t('services.Refurbishment')} |
            {t('services.BusinessMaximization')} | {t('services.Management')} |
            {t('services.Coordination')} | {t('services.AfterSales')} |
            {t('services.Planning')}
          </h2>
        </div>
        <p>{t('services.TheTarget')}</p>
        <p>{t('services.ADeep')}</p>
        <p>{t('services.WeWill')}</p>
        <p>{t('services.WeCoordinate')}</p>
      </SEO>
    </div>
  )
}

export default Services
