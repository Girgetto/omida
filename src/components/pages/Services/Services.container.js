import { connect } from 'react-redux'
import Services from './Services.styled';

export default connect()(Services);
