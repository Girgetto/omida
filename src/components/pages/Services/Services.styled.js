import styled from 'styled-components'

import Services from './Services.component'
import background from '../../../assets/img/services4.jpg'

export default styled(Services).attrs({})`
  .header {
    position: relative;
    background-image: url(${background});
    background-repeat: no-repeat;
    background-size: cover;
    overflow: hidden;
    height: 400px;
    display: flex;
    justify-content: center;
    align-items: center;
    text-align: center;
    #overlay {
      position: absolute;
      width: 100%;
      height: 100%;
      top: 0;
      left: 0;
      right: 0;
      bottom: 0;
      background-color: rgba(0, 0, 0, 0.5);
      z-index: 1;
    }
    p {
      color: white;
      font-weight: 500;
      font-size: 26px;
      width: 80%;
      z-index: 2;
    }
  }

  .content {
    margin: 0 auto;
    width: 60%;

    .titles {
      letter-spacing: 0.1rem;
    }
  }
`
