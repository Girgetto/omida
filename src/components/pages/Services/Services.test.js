import React from 'react';
import { render } from '@testing-library/react';
import { ParallaxProvider } from 'react-scroll-parallax'

import Services from './Services.component';

const props = {
  className: 'className',
};

describe('<Services />', () => {
  const { container } = render(
    <ParallaxProvider>
      <Services {...props} />,
    </ParallaxProvider>
  );

  it('match snapshot', () => {
    expect(container).toMatchSnapshot();
  });
});
