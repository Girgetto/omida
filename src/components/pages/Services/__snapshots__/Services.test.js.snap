// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Services /> match snapshot 1`] = `
<div>
  <div
    class="className"
  >
    <h2>
      services.Intro
    </h2>
    <p>
      services.IntroText
    </p>
    <div
      class="titles"
    >
      <h3>
        services.LuxuryRealEstate
         | 
        services.BusinessAdvisory
         | 
        services.TechnicalAdvice
         | 
        services.Refurbishment
         | 
        services.BusinessMaximization
         | 
        services.Management
         | 
        services.Coordination
         | 
        services.AfterSales
         |
        services.Planning
      </h3>
    </div>
    <p>
      services.TheTarget
    </p>
    <p>
      services.ADeep
    </p>
    <p>
      services.WeWill
    </p>
    <p>
      services.WeCoordinate
    </p>
    <img
      alt="door"
      src="services4.jpg"
    />
  </div>
  ,
</div>
`;
